.github/workflows/CI.yml

```diff
@@ -0,0 +1,29 @@
+name: Tests
+
+permissions: read-all
+
+on:
+  push:
+    branches:
+  pull_request:
```

Comment:
suggestion: Unless you want to specify branches, I think you can simplify that to:
```yaml
on: [push, pull_request]
```

Reply:
I want to specify branches but I forgot to update it when I moved the PR out of the draft mode.